### DISCUSSION TITLE
有什么方法可以把系统複制到新盒子?

### DISCUSSION THREAD

**Discussion by jackcctse** [ophub/amlogic-s9xxx-armbian#1985] [Amlogic] [answered]
大家好, 请问有什么方法可以把系统複制到新盒子?

现有盒子是CM311-1a-YST, 系统是Armbian_bookworm, 5.15内核, 已刷到emmc。想把整个系列複制到MGV3000。

armbian-ddbr指令可以备份系统, 那是不是可以还原到其他盒子? 有没有配置文件要修改去适配新盒子?

**Comment by ophub** [comment]
armbian-ddbr 选b是备份，选r是还原

**Comment by jackcctse** [comment]
> armbian-ddbr 选b是备份，选r是还原


嗯嗯, 了解的。我意思是, CM311-1a-YST的备份, 要怎样更改才能在MGV3000上运行? 笔竟他们的dtb是不同的。

**Comment by jackcctse** [comment, marked as answer]
我自己解答。armbian-ddbr還原到其他盒子后，修改/boot/eEnv.txt裡面的dtb。如我的MGV3000就是meson-gxl-s905l2-x7-5g.dtb。

`LINUX=/zImage`
`INITRD=/uInitrd`
`FDT=/dtb/amlogic/meson-gxl-s905l2-x7-5g.dtb`
`APPEND=root=UUID=f3cc12b8-759a-4ae2-b852-904755f2da39 rootflags=data=writeback rw rootfstype=ext4` `console=ttyAML0,115200n8 console=tty0 no_console_suspend consoleblank=0 fsck.fix=yes fsck.repair=yes net.ifnames=0` `max_loop=128 cgroup_enable=cpuset cgroup_memory=1 cgroup_enable=memory swapaccount=1`


註：只适用 5.15内核
